lib/task-data/base-tasks/linux-bootstrapper.js

```diff
@@ -11,8 +11,6 @@ module.exports = {
         'initrdFile',
         'kernelUri',
         'initrdUri',
-        'basefs',
-        'overlayfs',
```

Comment:
FYI @yyscamper, I assume these are in/going in the task schemas? Will want to make them optional values.